setup.py

```diff
@@ -114,30 +114,34 @@ def run(self):
         'invenio_i18n.translations': [
             'messages = rero_ils',
         ],
+        # Admin menu ordering
```

Comment:
Can you add a `TODO:` prefix in the comment such as:
```
# TODO: remove this dirty hack when the following PR (https://github.com/flask-admin/flask-admin/pull/1798) will be integrated
```

Reply:
done